src/main/java/utils/Utils.java

```diff
@@ -0,0 +1,20 @@
+package utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Utils {
+    public static void waitInSeconds (int seconds) {
+    try {
+        Thread.sleep(seconds * 1000);
+    }
+    catch (Exception e){
+        System.err.println(e.getMessage());
+    }
+
+    }
+    public static void scrollToElement(WebElement element, WebDriver driver) {
+        ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", element);
+    }
```

Comment:
It would be more clean and readable if you leave an empty line before any new method declaration.

Reply:
done